php/php.editor/src/org/netbeans/modules/php/editor/verification/PHP55UnhandledError.java

```diff
@@ -124,11 +124,15 @@ public void visit(StaticConstantAccess node) {
             if (CancelSupport.getDefault().isCancelled()) {
                 return;
             }
-            Identifier constant = node.getConstantName();
-            if (constant != null) {
-                String constantName = constant.getName();
-                if ("class".equals(constantName.toLowerCase())) { //NOI18N
-                    createError(constant);
+            if (node.isDynamicName()) {
+                super.visit(node);
+            } else {
+                Identifier constant = node.getConstantName();
+                if (constant != null) {
+                    String constantName = constant.getName();
+                    if ("class".equals(constantName.toLowerCase())) { //NOI18N
```

Comment:
Nitpick: I believe we should use some locale in `toLowerCase()` call.

Reply:
Yes, I'll fix it. Thanks!